src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -101,33 +99,25 @@ public static Scope CreateOutboundHttpScope(Tracer tracer, string httpMethod, Ur
                     return null;
                 }
 
-                string resourceUrl = requestUri != null ? UriHelpers.CleanUri(requestUri, removeScheme: true, tryRemoveIds: true) : null;
-                string httpUrl = requestUri != null ? UriHelpers.CleanUri(requestUri, removeScheme: false, tryRemoveIds: false) : null;
-
                 tags = new HttpTags();
-
                 string serviceName = tracer.Settings.GetServiceName(tracer, ServiceName);
-                scope = tracer.StartActiveWithTags(OperationName, tags: tags, serviceName: serviceName, spanId: spanId);
-
-                var span = scope.Span;
-
-                span.Type = SpanTypes.Http;
-                span.ResourceName = $"{httpMethod} {resourceUrl}";
-
-                tags.HttpMethod = httpMethod?.ToUpperInvariant();
-                tags.HttpUrl = httpUrl;
-                tags.InstrumentationName = IntegrationRegistry.GetName(integrationId);
-
-                tags.SetAnalyticsSampleRate(integrationId, tracer.Settings, enabledWithGlobalSetting: false);
+                var scope = tracer.StartActiveWithTags(OperationName, tags: tags, serviceName: serviceName, spanId: spanId);
+                Convention.Get(tracer.Settings).OutboundHttpScopeConvention.Apply(new OutboundHttpArgs
```

Comment:
Instead of invoking it as a `Get` method, it seems better to set a `Convention` property on the `Tracer`.

Reply:
Addressed here: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/63/commits/c81a57ae52909d614a7bd7b58b83efb99ec2385d